workflow/react-project/src/pages/SearchPage/CocktailCard/index.jsx

```diff
@@ -10,7 +10,6 @@ const CocktailCard = ({ name, picture }) => {
       <div className="cocktail-card-photo-wrapper">
         <img className="cocktail-card-photo" src={picture} />
       </div>
-
       <p className="cocktail-card-name">{name}</p>
       <div className="d-flex">
         <Tooltip text="Contain Alcohol">
```

Comment:
you have to create common component for different cocktail types (alcoholic, non alcoholic),  which will get `isAlcoholic` and return icon with tooltip.
Each cocktail has field - `strAlcoholic` (it's value are `"Non alcoholic"`/"Alcoholic"), you can map cocktails before writing to redux store and write new field `isAlcoholic: true/false`.

Then in the same way you can add other types (showing by diff icon-tooltip). For ex. each cocktail has it's own glass, and you can specify diff icons for diff glasses (`strGlass `)